src/main/resources/com/xebialabs/deployit/ci/Credential/config.jelly

```diff
@@ -1,34 +1,44 @@
 <?jelly escape-by-default='true'?>
-<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:c="/lib/credentials" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form">
-<f:block>
-    <f:entry title="${%Name}" field="name">
-        <f:textbox/>
-    </f:entry>
-    <f:optionalBlock field="secondaryServerInfo" title="Use non-default XL Deploy Server" checked="${instance.showSecondaryServerSettings()}">
-        <f:entry title="${%Server Url}" field="secondaryServerUrl">
-            <f:textbox/>
-        </f:entry>
-        <f:entry title="${%Proxy Url}" field="secondaryProxyUrl">
-            <f:textbox/>
-        </f:entry>
-    </f:optionalBlock>
-    <f:radioBlock inline="true" name="useGlobalCredential" value="false" title="Use username/password" checked="${!instance.showGolbalCredentials()}">
-        <f:entry title="${%Username}" field="username">
-            <f:textbox/>
-        </f:entry>
-        <f:entry title="${%Password}" field="password">
-            <f:password/>
-        </f:entry>
-        <f:validateButton title="${%Test Credential}" with="deployitServerUrl,deployitClientProxyUrl,username,password,secondaryServerUrl,secondaryProxyUrl"
-                          method="validateUserNamePassword"/>
-    </f:radioBlock>
-    <f:radioBlock inline="true" name="useGlobalCredential" value="true" title="Use stored credentials" checked="${instance.showGolbalCredentials()}">
-        <f:entry title="${%Credentials}" field="credentialsId">
-            <c:select/>
-        </f:entry>
-        <f:validateButton title="${%Test Connection}" with="deployitServerUrl,deployitClientProxyUrl,secondaryServerUrl,secondaryProxyUrl,credentialsId"
-                          method="validateCredential"/>
-    </f:radioBlock>
-    <f:description><br/></f:description>
-</f:block>
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:c="/lib/credentials"
+         xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form">
+    <l:isAdmin>
```

Comment:
I think my original advice has been wrong as the class name confused me. This is not actually a "credential" implementation for https://plugins.jenkins.io/credentials/ itself, but _holds_ credentials. This UI, AFAICT, is only shown in the global Jenkins configuration, where you can assume all users have Overall/Administer, so we do not need to hide UI.

(And FWIW this would hide the entire UI, rather than just the "Test" button, which was what I was trying to recommend.)

If this is also shown elsewhere, it may need to be done differently.